Subject: [PATCH] swaybg: make librt optional on iOS

iOS has clock_gettime() in libc and no separate librt.

diff -ruN -U0 --exclude=*.bak a/meson.build b/meson.build
--- a/meson.build	2026-02-25 00:00:00
+++ b/meson.build	2026-07-06 00:00:00
@@ -30 +30 @@
-rt = cc.find_library('rt')
+rt = cc.find_library('rt', required: false)
